application-accelerator/install-app-acc.hbs.md

```diff
@@ -109,26 +109,37 @@ To install Application Accelerator:
     >**Note** For clusters that do not support the `LoadBalancer` service type, override the default
     >value for `server.service_type`. For example:
 
-      >```yaml
+      > ```yaml
       > server:
       >   service_type: "ClusterIP"
       >   watched_namespace: "accelerator-system"
       > samples:
       >   include: true
-      >```.
+      > ```
+    
+    >**Note** When working instead with a `tap-values.yaml` file, ensure that the top level parameter is `accelerator:`. Using the example above as reference:
+      > ```yaml
+      > accelerator:
```

Comment:
using this `accelerator` entry won't work when installing the accelerator package individually - the properties must be at the top level in the yaml file

Reply:
Got it - so this addition should be entirely removed. Is that right?